snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs

```diff
@@ -35,6 +35,8 @@ static async Task Main(string[] args)
                 }
                 allTasks.Remove(finished);
             }
+            Juice oj = PourOJ();
+            Console.WriteLine("oj is ready");
```

Comment:
```suggestion
            Console.WriteLine("Orange juice is ready");
```

Reply:
The docs page has oj is ready in several other places. Should we change others as well or just this one?

Reply:
I didn't know about other occurrences. With them, I think the consistency should be maintained. The simplest way to do that is to ignore my suggestion and leave `oj` as it is.

Reply:
Thanks. Resoling the comment